courses/index.md

```diff
@@ -3,6 +3,72 @@ id: index
 title: Index
 sidebar_label: Index
 slug: /
+hide_table_of_contents: true
 ---
 
-# Hello
+# Offered courses
+
+### Requirements Legend
+
+❤️Bachelor of Computer Science (General)<br/>
+💙Bachelor of Computer Science (Honours)<br/>
+💛Bachelor of Computer Science (Honours Applied Computing)<br/>
+💚Bachelor of Science (Honours Computer Science with Software Engineering Specialization)<br/>
+💜Bachelor of Commerce (Honours Business Administration and Computer Science)<br/>
+🖤Bachelor of Mathematics (Honours Mathematics and Computer Science)<br/>
```

Comment:
I thought we once talked about changing this heart to some other representations, isn't it? Do we do that in new PR?

Reply:
it looks exactly like the old one but on the new page. I thought it was best to put a quick port up first and then work on the new features.

Reply:
Ok sounds good